gasSchedules/gasScheduleV9.toml

```diff
@@ -0,0 +1,844 @@
+[BuiltInCost]
```

Comment:
Identical to the one on testnet:
 - https://github.com/multiversx/mx-chain-testnet-config/blob/master/gasSchedules/gasScheduleV9.toml

:heavy_check_mark: